js/components/controlBar.js

```diff
@@ -219,6 +221,56 @@ var ControlBar = React.createClass({
     this.props.controller.state.focusedControl = null;
   },
 
+  /**
+   * Will handle the keydown event when the controlBar is active and it will restrict
+   * tab navigation to elements that are within it when the player is in fullscreen mode.
+   * @private
+   * @param {Object} evt Keydown event object.
+   */
+  handleControlBarKeyDown: function(evt) {
+    if (
+      evt.key !== CONSTANTS.KEY_VALUES.TAB
+      || !this.props.controller.state.fullscreen
+      || !this.domNode
+      || !evt.target
+    ) {
+      return;
+    }
+    // Focusable elements on the control bar (this.domNode) are expected to have the
+    // data-focus-id attribute
+    var focusableElements = this.domNode.querySelectorAll('[data-focus-id]');
+
+    if (focusableElements.length) {
```

Comment:
If focusableElements is empty, can we have a debug log or is it valid to not have any elements?  Want to make sure if someone is messing with css or something else and accidentally erases all these elements, they have a warning.

Reply:
Not very likely since these are elements like the scrubber bar, play/pause button, but I can add log